packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -542,6 +518,38 @@ export function makeRumPublicApi(
     }),
 
     stopSessionReplayRecording: monitor(() => recorderApi.stop()),
+
+    addDurationVital: monitor(
+      (name: string, options: { startTime: number; duration: number; context?: object; description?: string }) => {
```

Comment:
Now that it is in `RumPublicApi` interface, you don't need to specify the types here:

```suggestion
      (name, options) => {
```